src/styles/typography.scss

```diff
@@ -3,6 +3,7 @@
 .subtitle-1,
 .subtitle-2 {
   font-weight: bold;
+  color: $primaryColor;
```

Comment:
Eigentlich sollte die primaryColor automatisch standardmäßig gesetzt sein. Gibt es einen Fall, in dem das nicht so ist?

Reply:
Die Farbe bei den Headlinen und Subtitles ist nur nach dem Hinzufügen von  color: $primaryColor zu sehen ist, also sie nicht wirklich automatisch gesetzt für das ganze Body :/

Reply:
Habe die globalen Vuetify Variablen auf unsere Schriftfarbe angepasst, dann muss das nicht immer wieder überschrieben werden.